src/dmd/frontend.h

```diff
@@ -1703,7 +1703,7 @@ struct ParameterList
     Array<Parameter*>* parameters;
     StorageClass stc;
     VarArg varargs;
-    ParameterList(Array<Parameter*>* parameters, VarArg varargs = cast(VarArg)cast(ubyte)0u, StorageClass stc = 0);
+    ParameterList(Array<Parameter*>* parameters, VarArg varargs = (VarArg)0u, StorageClass stc = 0);
```

Comment:
Why is this still a `cast`, but `STC` is displayed correctly ?

Reply:
`StorageClass` is a typedef, not an enum.